### PR TITLE
brett's try

### PR DESCRIPTION
**Pull request by bsw975**
comment for pull req

### CODE REVIEW

File: brettdex.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Making webpages is fun</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+    <script src="main.js"></script>
+</head>
+<body>
+
```
Comment (hankzimmer7) [inline brettdex.html]: Need to add more in the body

**Review by hankzimmer7 — COMMENTED** [review]
The file looks good

**Review by hankzimmer7 — COMMENTED** [review]
Looks good!

File: brettdex.html (hunk 1 above)
Comment (bsw975) [reply brettdex.html]: i like the body just the way it is

**Review by hankzimmer7 — APPROVED** [review]

**Pull request merged**